src/OpenDB/test/unitTestsPython/README.md

```diff
@@ -2,27 +2,18 @@
 
 ### Running tests
 
-For running the python unit tests you will need to install first *testtools* and *unittest-parallel* which enables concurrent unit testing
+For running the python unit tests you will need to install first *testtools*  which enables unit testing
```

Comment:
first install
Also there's an extra space

Reply:
Is the runtime of these tests long enough to warrant adding a dependency?

Reply:
the dependency on testtools is not for parallel unit testing. It's the testing framework used in these tests.

Reply:
In that case why not use the default one https://docs.python.org/3/library/unittest.html

Reply:
Excuse my lack of concentration, yes testtools are for parallel and we use them along unittest. No the tests are not that long so we can drop it.